sys/net/gnrc/netif/gnrc_netif_ieee802154.c

```diff
@@ -95,7 +94,9 @@ static gnrc_pktsnip_t *_recv(gnrc_netif_t *netif)
             gnrc_pktbuf_release(pkt);
             return NULL;
         }
-        if (!(state->flags & NETDEV_IEEE802154_RAW)) {
+        netopt_enable_t raw = NETOPT_DISABLE;
+        dev->driver->get(dev, NETOPT_RAWMODE, &raw, sizeof(raw));
+        if (!(raw == NETOPT_ENABLE)) {
```

Comment:
Why not `!=`?

Reply:
Let me change that conditional instead of trying to answer your question :)

Reply:
Any preference between `raw != NETOPT_ENABLE` or `raw == NETOPT_DISABLE`?